src/devices/mixer.cpp

```diff
@@ -312,9 +316,9 @@ void Mixer::process_MMStatusMessage(std::shared_ptr<const Telegram> telegram) {
     // 0x21 is position 2. 0x20 is typically reserved for the WM10 switch module
     // see https://github.com/proddy/EMS-ESP/issues/270 and https://github.com/proddy/EMS-ESP/issues/386#issuecomment-629610918
     hc_ = device_id() - 0x20 + 1;
-    changed_ |= telegram->read_value(flowTemp_, 1);         // is * 10
+    changed_ |= telegram->read_value(flowTempLowLoss_, 0); // T0
```

Comment:
For MM10 this is the setpoint calculated by thermostat and send to the mixer, republished here. 
BTW: What is a low loss header?

Reply:
low loss header in german is `hydraulische weiche`, this comes from the MM100 documentation. For the MM100 it's a sensor directly attached to the module.
If MM10 and MM100 use the same id for different values, we should probably differentiate them with different names

Reply:
Yes, i found in MM100 manual that it is a sensor T0. But this single byte value is not T0, measured values are always 16bit. Also Norberts list says for MM100 this is "Requested Flow temperature". I change it back for all mixers to FlowSetTemp and we have to search where T0 is published. It's not in 2D7.. telegrams. MM10 has only a single sensor named FV.

Reply:
ok, thanks for the info. In my setup, it's exactly the temperature from the low loss header. Will than have to find the actual sensor value in the MM100 telegrams. Already looking forward going through Norberts docu again :D